src/encoding.md

```diff
@@ -688,6 +689,53 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-mime-from-string]: #ex-mime-from-string
+<a name="ex-mime-from-string"></a>
+## Get MIME type from string
+
+[![mime-badge]][mime] [![cat-encoding-badge]][cat-encoding]
+
+The following example shows how to parse a [`MIME`] type from a string using the [mime] crate. You can handle a possible [`FromStrError`] by providing a default [`MIME`] type in an `unwrap_or` clause.
+
+```rust
+# #[macro_use]
```

Comment:
In this example there is no error handling (you are demonstrating how to explicitly handle the error in fact) so there is no need for the error_chain crate in this example.  You can remove lines 701-703,705-711, 733,736.